modules/core/include/opencv2/core.hpp

```diff
@@ -352,7 +352,7 @@ result of an incorrect sign in the case of overflow.
 @param src2 second input array or a scalar.
 @param dst output array that has the same size and number of channels as the input array(s); the
 depth is defined by dtype or src1/src2.
-@param mask optional operation mask - 8-bit single channel array, that specifies elements of the
+@param mask optional operation mask - CV_8U or CV_Bool single channel array, that specifies elements of the
```

Comment:
why not add CV_8S as yet another option, since we are doing changes anyway

Reply:
Done